netbox_cmdb/netbox_cmdb/api/snmp/serializers.py

```diff
@@ -0,0 +1,19 @@
+"""Route Policy serializers."""
+
+from rest_framework.serializers import ModelSerializer
+
+from netbox_cmdb.models.snmp import SNMP, SNMPCommunity
+
+
+class SNMPCommunitySerializer(ModelSerializer):
+
+    class Meta:
+        model = SNMPCommunity
+        fields = ["name", "community", "type"]
+
+
+class SNMPSerializer(ModelSerializer):
+
+    class Meta:
+        model = SNMP
+        fields = ["community_list", "location", "contact", "device"]
```

Comment:
in the other models we uses `fields = "__all__"`
related to my other comment

Reply:
in some of them, but not all :)  I didn't copy/paste the good one :) :)